NotificationBanner.xcodeproj/project.pbxproj

```diff
@@ -310,6 +326,7 @@
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
 				IPHONEOS_DEPLOYMENT_TARGET = 9.3;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
+				OTHER_SWIFT_FLAGS = "-DCARTHAGE_CONFIG";
```

Comment:
Note that the important project file change is here, I somewhat inadvertently included the change for importing the frameworks (was just getting it to build).